routes/web.php

```diff
@@ -11,33 +22,58 @@
 |
 */
 
-Route::get('/', function () {
-    return view('welcome');
-});
+Route::get(
+    '/', function () {
+        return view('welcome');
+    }
+);
```

Comment:
Why modify item not related to current work ?

Reply:
I have edited the content